src/util/tstring.hpp

```diff
@@ -5,8 +5,10 @@
 
 #ifdef _UNICODE
 using tstring = std::wstring;
+using tstring_view = std::wstring_view;
```

Comment:
I think this should be a new header, without the heavy dependency on `<string>`

Reply:
ok will move to new header.